references/classification/train.py

```diff
@@ -507,6 +511,7 @@ def get_args_parser(add_help=True):
         "--ra-reps", default=3, type=int, help="number of repetitions for Repeated Augmentation (default: 3)"
     )
     parser.add_argument("--weights", default=None, type=str, help="the weights enum name to load")
+    parser.add_argument("--backend", default="PIL", type=str, help="PIL or tensor - case insensitive")
```

Comment:
IIRC, there is a callback option here that we could use to call `.lower()`. Even if not, we should probably do this during parsing or in the main script rather inside the transforms. Optimally, we would also check the values here to avoid doing this only at the transform level.

Reply:
I prefer doing this at the lower level, those transforms can still be called on their own outside of the reference scripts

Reply:
Nit: Should we still have this `.lower()` call here to avoid doing it in the main script? I mean, right now there is only one occurrence so it doesn't really matter. However, if we have multiple ones in the future, we need to call it multiple times instead of one.